client/src/containers/projects/form/utils.ts

```diff
@@ -171,89 +192,64 @@ export default function CreateCustomProject() {
           })),
       },
     );
-
-  const methods = useForm<CreateCustomProjectForm>({
-    resolver: zodResolver(CreateCustomProjectSchema),
-    defaultValues: {
-      projectName: "",
-      activity: ACTIVITY.CONSERVATION,
-      ecosystem: ECOSYSTEM.SEAGRASS,
-      countryCode: countryOptions?.[0]?.value,
-      projectSizeHa: 20,
-      carbonRevenuesToCover: CARBON_REVENUES_TO_COVER.OPEX,
-      initialCarbonPriceAssumption: 1000,
-      parameters: {
-        emissionFactorUsed: EMISSION_FACTORS_TIER_TYPES.TIER_1,
-        lossRateUsed: LOSS_RATE_USED.PROJECT_SPECIFIC,
-        projectSpecificLossRate: -35,
-        projectSpecificEmission: PROJECT_SPECIFIC_EMISSION.ONE_EMISSION_FACTOR,
-        projectSpecificEmissionFactor: 0,
-        emissionFactorSOC: 0,
-        emissionFactorAGB: 0,
-        // @ts-expect-error fix later
-        plantingSuccessRate: 80,
-      },
-      assumptions: {
-        baselineReassessmentFrequency: undefined,
-        buffer: undefined,
-        carbonPriceIncrease: undefined,
-        discountRate: undefined,
-        projectLength: undefined,
-        restorationRate: undefined,
-        verificationFrequency: undefined,
+  const { data: project } = client.customProjects.getCustomProject.useQuery(
+    queryKeys.customProjects.one(id).queryKey,
+    {
+      params: { id: id ?? "" },
```

Comment:
This is a bit weird, why would the query for getting a project have a conditional `id` param?